templates/taxi/driver_list.html

```diff
@@ -1,17 +1,24 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Driver list</h1>
-  {% if driver_list %}
-    <ul>
-      {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
+  <h1>Driver List</h1>
+  {% if object_list %}
+    <table>
+      <tr>
+        <th>Username</th>
+        <th>Full Name</th>
+      </tr>
+      {% for driver in object_list %}
+        <tr>
+          <td>
+            <a href="{% url 'taxi:driver-detail' driver.pk %}">{{ driver.username }}</a>
```

Comment:
Ensure that the URL name 'taxi:driver-detail' used here matches the name defined in your `urls.py` for the driver detail view. This is important for the links to function correctly.